sys/dev/etherswitch/e6000sw/e6000sw.c

```diff
@@ -261,6 +262,8 @@ e6000sw_probe(device_t dev)
 		    device_get_unit(sc->dev), "is8190", &is_6190);
 	if (is_6190 != 0)
 		sc->swid = MV88E6190;
+	if (is_6190x != 0)
```

Comment:
Where is is_6190x variable initialized? Searching "is_6190x" only gave me two results

Reply:
I am not a huge fan of increasing number of `if` statements to support additional switch variants but I feel it's a good tradeoff to keep the change small and backwards compatible.